pkg/transforms/policyreport.go

```diff
@@ -51,11 +55,13 @@ func PolicyReportResourceBuilder(pr *PolicyReport) *PolicyReportResource {
 	node.Properties["apiversion"] = gvk.Version
 	node.Properties["apigroup"] = gvk.Group
 
+	isKyverno := pr.Labels[managedByLabel] == "kyverno"
+
 	// Filter GRC sourced policy violations from node results
 	// Policy details are displayed elsewhere in the UI, displaying them in the PR will result in double counts on pages
 	results := []ReportResults{}
 	for _, result := range pr.Results {
-		if result.Source == "insights" {
+		if result.Source == "insights" || (isKyverno && (result.Result == "fail" || result.Result == "error")) {
```

Comment:
What about `warn`?

Reply:
@yiraeChristineKim I don't think I'd consider it a rule violation if it's a warning, though I haven't seen a warning provided by Kyverno. Even for an audit only policy, it just puts "fail" for a failed audit.